src/Glpi/Kernel.php

```diff
@@ -391,9 +391,9 @@ private function loadPluginsServices(ContainerBuilder $container)
      * Else, instanciate a non configure DB to be able to compile container.
      * Returns global variable reference that will be updated if a new connection is established.
      *
-     * @return AbstractDatabase
+     * @return Glpi\Database\AbstractDatabase
      */
-    private function getDbInstance(): AbstractDatabase
+    private function getDbInstance(): Database\AbstractDatabase
```

Comment:
Maybe adding a use statement would be a better solution.

Reply:
Done